iroha/src/account.rs

```diff
@@ -5,70 +5,149 @@ use std::collections::BTreeMap;
 #[derive(Debug, Clone, Encode, Decode)]
 pub struct Account {
     pub id: Id,
-    pub assets: BTreeMap<Id, Asset>,
+    pub assets: BTreeMap<<Asset as Identifiable>::Id, Asset>,
+    signatories: Vec<PublicKey>,
 }
 
 impl Account {
-    pub fn new(account_id: Id) -> Self {
+    pub fn new(account_name: &str, container: &str, public_key: PublicKey) -> Self {
         Account {
-            id: account_id,
+            id: Id::new(account_name, container),
             assets: BTreeMap::new(),
+            signatories: vec![public_key],
         }
     }
 }
 
+/// Identification of an Account. Consists of Account's name and Domain's name.
+///
+/// # Example
+///
+/// ```
+/// use iroha::account::Id;
+///
+/// let id = Id::new("user", "company");
+/// ```
+#[derive(Clone, Debug, PartialEq, PartialOrd, Ord, Eq, std::hash::Hash, Encode, Decode)]
+pub struct Id {
+    pub name: String,
+    pub container: String,
```

Comment:
why call this `container` and not `domain`?

Reply:
@takemiyamakoto - I made an additional issue to solve current problem with Iroha model https://soramitsu.atlassian.net/browse/RH2-88

Reply:
Both asset and account definitions are namespaces by domain, so it makes sense to have under domain I think. You can have the same asset name, e.g., XOR, under multiple domains, such as ethereum#xor and sora#xor.